package.json

```diff
@@ -71,8 +71,10 @@
     "start": "react-scripts start",
     "init": "node scripts/init.js",
     "build": "react-scripts build",
-    "lint": "prettier --write \"{src,scripts,functions,configs,reporting}/**/*.{js,jsx,ts,tsx,json,css,scss,md}\"",
-    "lint:src": "eslint src --ext .js,.jsx,.ts,.tsx",
+    "lint": "npm run prettier:check && npm run eslint:src",
+    "eslint:src": "eslint src --ext .js,.jsx,.ts,.tsx",
```

Comment:
Let's also add `eslint:fix` here for discoverability

Reply:
done